src/Elasticsearch.Net/Connection/InMemoryConnection.cs

```diff
@@ -1,8 +1,8 @@
-// Licensed to Elasticsearch B.V under one or more agreements.
-// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
-// See the LICENSE file in the project root for more information
-
-﻿using System;
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+ using System;

```

Comment:
:sadpanda:

Reply:
Rider is transforming `ZWNBSP` (as checked in) to spaces on linux. 

Will try and clean these up today.